src/assets/icons/icon-box.svg

```diff
@@ -0,0 +1,12 @@
+<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
unfortunately, we have to go back through and add the  `Icon` class to all the **svg** elements and the add the `<iconName>-icon` id to the top level **g**(group tag) like how we have it in the uikit. https://github.com/kids-first/kf-uikit/blob/master/src/components/Icon/Icon.jsx

Otherwise we won't be able to uniformly style these like the other icons